src/neighbors.cc

```diff
@@ -1456,6 +1462,7 @@ peerCountMcastPeersDone(void *data)
     fake->mem_obj->request = nullptr;
     fake->unlock("peerCountMcastPeersDone");
     delete psstate;
+    CodeContext::Reset();
```

Comment:
This call is not required since the context will be cleaned soon in AsyncCallQueue::fire(). I decided to do this explicitly so that peerCountMcastPeersStart/peerCountMcastPeersDone were symmetrical.